waiter/test/waiter/util/utils_test.clj

```diff
@@ -118,7 +118,9 @@
     (testing "should convert empty map"
       (let [{:keys [body headers status]} (clj->json-response {})]
         (is (= 200 status))
-        (is (= {"content-type" "application/json"} headers))
+        (is (= {"content-type" "application/json"
```

Comment:
Ditto here.

Reply:
Introduced a new variable to hold the expected headers and left the assertions in the tests.